l2geth/consensus/clique/synchronizer/server.go

```diff
@@ -64,7 +67,14 @@ func (sync *Synchronizer) GetSchedulerAddr() (common.Address, error) {
 	if err != nil {
 		return common.Address{}, fmt.Errorf("create rpcClient failed: %s", err)
 	}
-	return seqContractInst.Scheduler(nil)
+	schedulerAddr, err := seqContractInst.Scheduler(nil)
+	for schedulerAddr.String() == "0x0000000000000000000000000000000000000000" && err == nil {
```

Comment:
We can use `common.Address{}.toString()` to replace `"0x0000000000000000000000000000000000000000"`

Reply:
> We can use `common.Address{}.toString()` to replace `"0x0000000000000000000000000000000000000000"`

changed.